packages/flutter_tools/test/general.shard/macos/xcode_test.dart

```diff
@@ -20,267 +19,356 @@ import '../../src/common.dart';
 import '../../src/context.dart';
 
 void main() {
-  ProcessManager processManager;
   Logger logger;
 
   setUp(() {
     logger = BufferLogger.test();
-    processManager = MockProcessManager();
   });
 
-  group('Xcode', () {
-    Xcode xcode;
-    MockXcodeProjectInterpreter mockXcodeProjectInterpreter;
-    MockPlatform platform;
-    FileSystem fileSystem;
+  // Work around https://github.com/flutter/flutter/issues/56415.
```

Comment:
Maybe for people who haven't read the issue you should explicitly call out that we shouldn't add to this group unless absolutely necessary